addon/templates/components/polaris-range-slider.hbs

```diff
@@ -0,0 +1,38 @@
+{{#polaris-labelled
+  id=id
+  label=label
+  error=error
+  action=labelAction
+  labelHidden=labelHidden
+  helpText=helpText
+}}
+  <div class={{rangeInputWrapperClassNames}} style={{rangeInputWrapperStyle}}>
+    <input
+      type="range"
+      class="Polaris-RangeSlider__Input"
+      id={{id}}
+      name={{id}}
+      min={{min}}
+      max={{max}}
+      step={{step}}
+      value={{value}}
+      disabled={{disabled}}
+      oninput={{action "handleChange"}}
```

Comment:
why not `onchange`?

Reply:
`onchange` only fired once at the end of dragging the slider handle, which meant that the fill on the slider, output value in the tooltip etc. didn't update during dragging. `oninput` seemed to fix that.